regulations/static/regulations/js/source/views/comment/comment-review-view.js

```diff
@@ -10,19 +10,39 @@ var CommentEvents = require('../../events/comment-events');
 
 var CommentReviewView = Backbone.View.extend({
   events: {
-    'click .comment-review-submit': 'submit'
+    'click #preview-button': 'preview',
```

Comment:
Can you change the IDs to classes? Ideally I'd like to have most selectors attached to classes (https://pages.18f.gov/frontend/css-coding-styleguide/specificity/).